### ISSUE TITLE
Multiple Individual Selection for scheduling vaccination is missing  

### ISSUE THREAD

**Issue by rahultilloo** [cowinapi/developer.cowin#371]
User is unable to select multiple individuals for scheduling vaccination. This feature was present there earlier but currently it's not present in the current application. Please check

**Comment by AdityaPimpalkar** [comment]
@rahultilloo You can add more beneficiaries when you are booking , right? I am guessing that's how you can book for multiple people - by passing their beneficiary Id in the body of the request

**Comment by nurav1997** [comment]
I think he is talking about the cowin self-care web app feature and not the API

**Comment by riyaznet** [comment]
Yes, this feature is much needed in the Cowin web app. Cowin team is requested to bring it back.

**Comment by rahultilloo** [comment]
@riyaznet  Yes, absolutely right. The feature was present earlier but in this new build it is not present.

@AdityaPimpalkar Sorry, I should have added that the Issue is related Main Cowin Website. User is able to add multiple beneficiaries at time of registration but unable to select them for same slot booking as the selection feature which was there earlier is now not working or removed.
